scripts/update.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
```

Comment:
A description comment of what this script does and why it's useful would be good.

Reply:
I assume it's to update this repo when there are changes to the separated SDK packages?

Reply:
Yes. I will add a few comments here.